src/App.svelte

```diff
@@ -0,0 +1,494 @@
+<script>
+// import { onMount } from "svelte";
+let value = 0;
+let counter = 1;
+let themeValue = 1;
+let storage = [0,0];
+let operator = false;
+let operatorWhich = "";
+let firstTime = false;
+let dotClicked = false;
+
+const updateTheme = (a) => {
+	themeValue = a.target.value;	
+}
+
+const number = (x) => {
+	
+	if (operator) {
+		value = x;
+	} else if(value.toString().length == 14) {
```

Comment:
It's better to make this check using >= instead of ==
If somehow it reaches length 15, it would never enter here.

Reply:
Changed